app/connectors/AddressLookupConnector.scala

```diff
@@ -55,4 +58,32 @@ class AddressLookupConnector @Inject() (http: HttpClient, config: FrontendAppCon
         throw e
     }
   }
+
+  def mkString(p: AddressLookup) =
+    List[String](p.addressLine1.getOrElse(""), p.addressLine2.getOrElse(""), p.addressLine3.getOrElse(""), p.addressLine4.getOrElse(""))
```

Comment:
@mbiczak I am not sure why this getOrLElse is need why cant you just flatten the option for none

Reply:
I still need the empty token rather then no token